mu-plugins/blocks/language-suggest/src/block.json

```diff
@@ -0,0 +1,24 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "wporg/language-suggest",
+	"version": "0.1.0",
+	"title": "Language Suggest",
+	"category": "widgets",
+	"icon": "admin-site-alt3",
+	"description": "A block for use across the whole wp.org network.",
+	"textdomain": "language-suggest",
+	"editorScript": "file:./index.js",
+	"viewScript": "file:./front.js",
+	"style": "file:./style.css",
+	"supports": {
+		"align": true,
+		"color": {
+			"background": true,
+			"text": true
+		},
```

Comment:
Can you add the link color too? Or remove this support for now, otherwise it's possible to create an unreadable link:

<img width="431" alt="Screen Shot 2022-08-01 at 6 33 27 PM" src="https://user-images.githubusercontent.com/541093/182256714-628a07c9-564e-4c81-8701-4706ee39a2e0.png">

Reply:
Added.